ext/easylogging-8.91/easylogging++.h

```diff
@@ -2249,6 +2255,30 @@ class Logger {
         return configured_;
     }
 
+    // no checks performed
+    // target must exist longer, than logger OR be removed
+    inline void addLogTarget(ILogTarget* target) {
+        customTargets_.push_back(target);
+    }
+
+    struct ShouldDelete {
```

Comment:
Would it be possible to make `ShouldDelete` private so as not to pollute public interface (maybe it would make a difference for various IDEs)?